### PR TITLE
bpo-23750: Document os-system, subprocess. Patch by Martin Panter.

### PR DESCRIPTION
**Pull request by uniocto**
https://bugs.python.org/issue23750

### CODE REVIEW

File: Doc/library/os.rst [hunk 1]
```diff
@@ -4211,12 +4211,12 @@ written in Python, such as a mail server's external command delivery program.
    the Standard C function :c:func:`system`, and has the same limitations.
    Changes to :data:`sys.stdin`, etc. are not reflected in the environment of
    the executed command. If *command* generates any output, it will be sent to
-   the interpreter standard output stream.
+   the interpreter standard output stream. The C standard does not
+   specify the meaning of the return value of C function, so the return
```
Comment (Fidget-Spinner) [inline Doc/library/os.rst]:
```suggestion
   specify the meaning of the return value of the C function, so the return
```

**Review by Fidget-Spinner — APPROVED** [review]
Awesome! Thank you for converting the patch to a PR :). Everything looks good (with just a few minor things below).

File: Doc/library/subprocess.rst [hunk 2]
```diff
@@ -1292,11 +1292,17 @@ Replacing :func:`os.system`
...
-   sts = call("mycmd" + " myarg", shell=True)
+   retcode = call("mycmd" + " myarg", shell=True)
 
 Notes:
 
 * Calling the program through the shell is usually not required.
+* The :func:`call` return value is encoded differently to that of
+  :func:`os.system`.
+
+* The :func:`os.system` function ignores SIGINT and SIGQUIT signals while
```
Comment (orsenthil) [inline Doc/library/subprocess.rst:1303]: Excellent. :+1:  The bug report used the term "blocked", the correct behavior is "ignored". 
Ref - https://man7.org/linux/man-pages/man3/system.3.html

**Review by orsenthil — APPROVED** [review]
LGTM.

**Comment by Fidget-Spinner** [comment]
@uniocto congrats on your first pull request getting merged into CPython! Welcome!

**Pull request merged**